Frameworks/AddressBook/ABAddressBook.mm

```diff
@@ -16,44 +16,89 @@
 
 #import <AddressBook/ABAddressBook.h>
 
-#import <StubReturn.h>
 #import "AssertARCEnabled.h"
+#import <StubReturn.h>
+#import <Starboard/SmartTypes.h>
+
+#import "_ABAddressBookManager.h"
+
+#import "UWP/InteropBase.h"
+#import "UWP/WindowsDevicesEnumeration.h"
+#import "initguid.h"
+
+// GUID to check Windows Contacts permissions.
+DEFINE_GUID(_ABAddressBookContactsGUID, 0x7D7E8402, 0x7C54, 0x4821, 0xA3, 0x4E, 0xAE, 0xEF, 0xD6, 0x2D, 0xED, 0x93);
 
 const CFStringRef ABAddressBookErrorDomain = static_cast<const CFStringRef>(@"ABAddressBookErrorDomain");
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 ABAddressBookRef ABAddressBookCreate() {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return ABAddressBookCreateWithOptions(NULL, NULL);
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 ABAddressBookRef ABAddressBookCreateWithOptions(CFDictionaryRef options, CFErrorRef* error) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    _ABAddressBookManager* addressBook = [[_ABAddressBookManager alloc] init];
+    if (addressBook.contactStore == nil) {
```

Comment:
absolutely but unnecessary <:D

---
In reply to: [73264467](https://github.com/Microsoft/WinObjC/pull/786#discussion_r73264467) [](ancestors = 73264467)